molecule_vagrant/modules/vagrant.py

```diff
@@ -52,77 +52,40 @@
 short_description: Manage Vagrant instances
 description:
   - Manage the life cycle of Vagrant instances.
-  - Supports check mode. Run with --check and --diff to view config difference,
-    and list of actions to be taken.
+  # - Supports check mode. Run with --check and --diff to view config difference,
+  #   and list of actions to be taken.
```

Comment:
afair, check mode and diff are not supported, what about just removing theses lines ?

Reply:
Removed in 6e20017.